src/features/bundlr/BundlrProfile.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Flex, Text } from '@chakra-ui/react';
+import { EthereumIcon } from 'components/icons/EthereumIcon';
+import { ethers } from 'ethers';
+import { useGetBalance } from 'features/embalm/stepContent/hooks/useGetBalance';
+import { useEthBalance } from 'hooks/useEthBalance';
+import { useEthPrice } from 'hooks/useEthPrice';
+import { useState } from 'react';
+import { BundlrInput } from './BundlrInput';
+
+export enum BundlrAction {
+  Deposit,
+  Withdraw,
+  Connect,
+}
+
+interface BundlrProfileProps {
+  action: BundlrAction;
+  onDeposit?: (amount: string) => void;
+  onWithdraw?: (amount: string) => void;
+  onConnect?: () => void;
+}
+
+export function BundlrProfile({ action, onDeposit, onWithdraw, onConnect }: BundlrProfileProps) {
+  const [amount, setAmount] = useState('');
+  const bundlrBalanceData = useGetBalance();
+  const bundlrBalance =
+    !bundlrBalanceData?.balance || bundlrBalanceData?.balance === ''
+      ? '--'
+      : Number(bundlrBalanceData?.balance).toFixed(2);
```

Comment:
I know this was in the designs also, we gotta show more than 2 decimals on ETH. right?

Reply:
I agree. We should probably show more like 4.